api/src/test/java/org/openmrs/module/fhirExtension/translators/ObsBasedDiagnosticReportTranslatorTest.java

```diff
@@ -192,6 +198,54 @@ public void givenFhirDiagnosticReport_WhenOnlyNotesObsArePresent_ShouldNotTransl
 		assertFalse(diagnosticReport.hasPresentedForm());
 	}
 	
+	@Test
+	public void givenFhirDiagnosticReport_WhenPerformerPresent_ShouldTranslatePerformerInFhirType() {
+		FhirDiagnosticReport fhirDiagnosticReport = new FhirDiagnosticReport();
+		Provider provider = new Provider();
+		Reference reference = new Reference("Practitioner");
+		reference.setReference("Practitioner/123");
+		provider.setUuid("123");
+		provider.setName("Test");
+		fhirDiagnosticReport.setPerformers(singleton(provider));
+		DiagnosticReport diagnosticReport = new DiagnosticReport();
+		when(diagnosticReportTranslator.toFhirResource(fhirDiagnosticReport)).thenReturn(diagnosticReport);
+		when(practitionerReferenceTranslator.toFhirResource(provider)).thenReturn(reference);
```

Comment:
We can also put an assertion for reference, as in where this value is been set:
assertEquals("Practitioner/123", result.getPerformers().get(0).getReference())

Reply:
added this assertion